src/AccessibilityInsights/MainWindowHelpers/AutoUpdate.cs

```diff
@@ -122,9 +123,15 @@ private UpdateDialog CreateUpgradeDialog(Uri releaseNotesUri, AutoUpdateOption u
         private async void DownLoadInstaller(IAutoUpdate autoUpdate, AutoUpdateOption updateOption)
         {
             UpdateResult result = UpdateResult.Unknown;
+
+            // The UAC prompt from the version switcher will appear behind the main window
+            // if it is topmost, we store, change, and restore the value in this method.
+            bool oldTopMost = Topmost;
```

Comment:
Could we call this variable previousTopmostSetting? I think that would be much more descriptive.

Reply:
Yes